pkg/tnf/handlers/deployments/deployments.go

```diff
@@ -52,17 +52,17 @@ type Deployments struct {
 }
 
 // NewDeployments creates a new Deployments tnf.Test.
-func NewDeployments(timeout time.Duration, namespace string) *Deployments {
+func NewDeployments(timeout time.Duration, namespace, resourceType string) *Deployments {
```

Comment:
resourceType string -> resourceType PodSetType

I think you redefined this in deployment.go, so it make more sense to use it